### DISCUSSION TITLE
Port 8443 not available

### DISCUSSION THREAD

**Discussion by Gene-W** [nextcloud/all-in-one#3305] [Questions] [answered]
Thank you for the [previous help](https://github.com/nextcloud/all-in-one/discussions/3289), things are coming along using Ubuntu server and I'm learning quite a lot along the way.

Using the server ip address, I can reach my AIO login page even from another computer.

However, in my router settings, I cannot forward port 8443 as I get the following error:
`Port 8443 is reserved by system and not available.`

Does this mean that I cannot use my domain name for my Nextcloud instance?

**Comment by szaimen** [comment, marked as answer]
Hi, just open the AIO interface internally via port 8080. It is also actually more secure if you dont open port 8080 or 8443 to the public internet. Also see https://github.com/nextcloud/all-in-one#which-ports-are-mandatory-to-be-open-in-your-firewallrouter

**Reply by Gene-W** [reply]
Very good, thank you.

Probably not directly AIO related, but do I configure Netplan in Ubuntu server to ensure that my AIO is configured to use a static IP?

**Reply by szaimen** [reply]
> Probably not directly AIO related, but do I configure Netplan in Ubuntu server to ensure that my AIO is configured to use a static IP?

Please ask this in an ubuntu forum.
